app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -23,6 +23,7 @@
     <string name="settings">设置</string>
     <string name="general">通用</string>
     <string name="notify_only_when_muted">仅在铃声静音时弹出通知</string>
+    <string name="auto_mute">自动静音</string>
```

Comment:
Please undo this. Translations are done via https://crowdin.com/project/mutereminder